src/Uno.UWP/UI/Notifications/ToastNotifier.Android.cs

```diff
@@ -0,0 +1,257 @@
+﻿#if __ANDROID__
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using System.Xml;
+
+namespace Windows.UI.Notifications
+{
+	public partial class ToastNotifier
+	{
+		private string _channelID = "PlatformUno"; 
+		private Android.App.NotificationManager _notificationManager;
+
+		public ToastNotifier()
+		{
+			_notificationManager = Android.App.NotificationManager.FromContext(Android.App.Application.Context);
+
+			if (Android.OS.Build.VERSION.SdkInt >= Android.OS.BuildVersionCodes.O)
+			{	// none of this strings would be visible for user
+				var channel = new Android.App.NotificationChannel(_channelID, "PlatformUnoChannel", Android.App.NotificationImportance.Default); ; // deprecated: Android.App.NotificationManager.ImportanceDefault
+				_notificationManager.CreateNotificationChannel(channel);
+			}
+
+		}
+
+		private string toastText2String(string toasttext)
+		{
+			// "When using the "ms-resource" prefix, the string identifier is referenced in the app's Resources.resw file."
+			if (!toasttext.StartsWith("ms-resource:"))
+			{
+				return toasttext;
+			}
+
+			toasttext = toasttext.Substring(12); // skipping prefix "ms-resource:"
+			var retVal = Windows.ApplicationModel.Resources.ResourceLoader.GetForCurrentView()?.GetString(toasttext);
+			if(retVal is null)
+			{	// we have no such string
+				return toasttext;
+			}
+			return retVal;
+		}
+
+		public void Show(ToastNotification notification)
+		{
+			var androidNotification = new Android.App.Notification();
+			androidNotification.Category = Android.App.Notification.CategoryMessage;
+
+			Android.App.Notification.Builder builder;
+
+			// if API < 11, then another version...
+			if (Android.OS.Build.VERSION.SdkInt >= Android.OS.BuildVersionCodes.O)
+			{
+				builder = new Android.App.Notification.Builder(Android.App.Application.Context, _channelID);
+			}
+			else
+			{
+#pragma warning disable CS0618 // Type or member is obsolete
+				builder = new Android.App.Notification.Builder(Android.App.Application.Context);

```

Comment:
What's the replaced API, as this one is obsolete ?

Reply:
See comment in code: `// if API < 11, then another version...` - newer version has two parameters, older version has one parameter.